### PR TITLE
Feature profile model

### PR DESCRIPTION
**Pull request by mayanksuchde**
### What this PR does (required):
Minor change: the userPhotosUrls were supposed to be an array of strings and not just string as there are more than one photos. I had made changes in the controller branch but forgot to change in this branch.
Sorry for any inconvenience

### CODE REVIEW

File: server/models/Profile.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ const profileSchema = new mongoose.Schema({
     ref: "User",
     required: true,
   },
-  userPhotoUrl: {
-    type: String,
+  userPhotosUrls: {
```
Comment (rajivtitus) [inline server/models/Profile.js:21]: Hey @mayanksuchde, are we planning to give the user an option to have single/multiple profile images?

Comment (mayanksuchde) [reply server/models/Profile.js:21]: I saw this image from the pdf and thought that was the case.

Screen Shot 2021-09-16 at 1 37 07 PM
c.

Comment (mayanksuchde) [reply server/models/Profile.js:21]: maybe we can choose a single index from the array and put it as a profile pic and the rest can be under the description.
Thoughts??

Comment (rajivtitus) [reply server/models/Profile.js:21]: I would suggest we have a separate field for `profilePhoto` (single) and another field for `galleryImages` (multiple).

Comment (rajivtitus) [reply server/models/Profile.js:21]: In the above case I've mentioned, we may need to make changes to the UI/create another component which we can use to upload gallery images.

Comment (mayanksuchde) [reply server/models/Profile.js:21]: > I would suggest we have a separate field for `profilePhoto` (single) and another field for `galleryImages` (multiple).

So in that case I would have to add 
2 separate routes and controllers; one to add profile image and one to add gallery Images, right?

Comment (mayanksuchde) [reply server/models/Profile.js:21]: and data type would be a String for a profile pic and array for galleryImages?

Comment (mayanksuchde) [reply server/models/Profile.js]: @rajivtitus Hey Rajiv I have made the changes in the model and added profilePic:String and galleryPics:[String] to the model. Will work on  routes and controller

**Review by rajivtitus — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**